patches/src/main/kotlin/app/revanced/patches/instagram/misc/signature/SignatureCheckPatch.kt

```diff
@@ -0,0 +1,19 @@
+package app.revanced.patches.instagram.misc.signature
+
+import app.revanced.patcher.patch.bytecodePatch
+import app.revanced.util.returnEarly
+
+@Suppress("unused")
+val signatureCheckPatch = bytecodePatch(
+    name = "Disable signature check",
+    description = "Disables the signature check that causes any modified app to crash on startup."
+) {
+    compatibleWith("com.instagram.android"("378.0.0.52.68"))
```

Comment:
```suggestion
    compatibleWith("com.instagram.android")
```

Perhaps remove the version constraint now that it should work on the latest versions

Reply:
Do we not worry about old versions?

Reply:
I'll let someone who knows better answer